src/uikit.cs

```diff
@@ -1560,7 +1560,7 @@ interface UIActivity {
 		[Export ("activityDidFinish:")]
 		void Finished (bool completed);
 
-		[Export ("activityCategory"), Static]
+		[Export ("activityCategory")]
```

Comment:
Thanks for the bug fix! While the change is valid in terms of the API usage, this adds an API breaking change. That means, our users will have compilation issues when they receive an update. 

The correctly way to do this is to add a deprecation, add to a small #if dance to ensure that when we have a chance to break the API, we can do so.

You can see an example here: https://github.com/xamarin/xamarin-macios/blob/main/src/uikit.cs#L9074

So we could need to do something like the following (please check that it compiles etc..)
```csharp

#if XAMCORE_5_0
		[Export ("activityCategory")]
#else
		[Obsolete ("Use the 'Category2' property instead.")]
		[Export ("activityCategory"), Static]
#endif
		UIActivityCategory Category { get; }

#if !XAMCORE_5_0
		[Export ("activityCategory")]
		UIActivityCategory Category2 { get; }
#endif
```

This way, we leave the old property and won't break our clients apps, they'll get a new warning and will fix accordingly. Next time we are allowed to change the API, we will get the correct behaviour.

Reply:
Thanks for the feedback - i will do the changes tomorrow

Reply:
i am open to suggestions on the naming 😀

Reply:
Perhaps it could be called `ActivityCategory` 😉